scanapi/scan.py

```diff
@@ -69,6 +72,13 @@ def write_report(results):
     reporter.write(results)
 
 
+def open_report_in_browser():
+    """Open the results file on a browser
```

Comment:
@renato04 if it's one line doc string it's good if the closing quotes are ends on the same line. That way static analysis won't fail too. 😄